docs/tutorial/category_theory/intro.lean

```diff
@@ -71,23 +70,18 @@ variables (f : W ⟶ X) (g : X ⟶ Y) (h : Y ⟶ Z)
 This says "let `C` be a category, let `W`, `X`, `Y`, `Z` be objects of `C`, and let `f : W ⟶ X`, `g
 : X ⟶ Y` and `h : Y ⟶ Z` be morphisms in `C` (with the specified source and targets)".
 
-Note two unusual things. Firstly, the typeclass `category C` is explicitly named as `𝒞` (in
-contrast to group theory, where one would just write `[group G]` rather than `[h : group G]`).
-Secondly, we have to explicitly tell Lean the universe where the morphisms live (by writing
-`category.{v} C`), because Lean cannot guess from knowing `C` alone.
+Note two unusual things. Firstly, we have to explicitly tell Lean the universe where the objects
+live (by writing `C : Type u`). We do not declare [category C], in contrast to group theory,
+where one would just write `[group G]`). Secondly, we also have to explicitly tell Lean the universe
+where the morphisms live (by writing `category.{v} C`), because Lean cannot guess from knowing `C`
+alone.
```

Comment:
```suggestion
Note that we need to explicitly tell Lean the universe that the morphisms live in, by writing `category.{v} C`, because Lean cannot guess this from `C` alone.
```

Reply:
I was about to add a note explaining when and why you need to add `.{v}` elsewhere when using the category theory API, but as I went through the library looking for examples, nearly all of them are now unnecessary! I will separately make a PR removing a huge fraction of them, and then try to come back to this tutorial.